doc/configure/mruby.html

```diff
@@ -269,6 +269,57 @@ <h3 id="logging-arbitrary-variable">Logging Arbitrary Variable</h3>
 
 
 
+<h3 id="using-redis">Using Redis</h3>
+
+<p>
+Since version 2.3, you can use redis from an mruby handler. 
+All redis methods are supported, however pub/sub calls are synchronous.
+</p>
+<p>
+
+</p>
+
+<div class="example">
+<div class="caption">Example. redis handler</div>
+<pre><code>
+listen:
+  port: 8080
+
+num-threads: 1
+hosts:
+  &quot;*&quot;:
+    paths:
+      /test:
+        mruby.handler: |
+          redis_host = ENV[&#39;REDIS_HOST&#39;] || &#39;localhost&#39;
+          redis_port = (ENV[&#39;REDIS_PORT&#39;] || &quot;6379&quot;).to_i
+          redis_db = (ENV[&#39;REDIS_DB&#39;] || &quot;1&quot;).to_i
+
+          redis = H2O::Redis.new(:host =&gt; redis_host, :port =&gt; redis_port, :db =&gt; redis_db)
+          redis.set(&quot;numberofthreads&quot;, 0).join
+          [ 
+            redis.set(&quot;users:1&quot;, &quot;kazuho&quot;), 
+            redis.incr(&quot;numberofthreads&quot;) 
+          ].map(&amp;:join)
+          #note: this will create one connection per h2o thread and per handler.
+
+          Proc.new do |env|
+            #parse params (but do not decode them)
+            params = Hash[env[&quot;QUERY_STRING&quot;].split(&quot;&amp;&quot;).map{|kv| kv.split(&quot;=&quot;)}] 
+            user_id = params[&quot;user_id&quot;]
+            user_name, threads = [
+              redis.get(&quot;users:#{user_id}&quot;),
+              redis.get(&quot;numberofthreads&quot;),
+            ].map(&amp;:join) #parallel calls
+            result_string = {&#39;name&#39; =&gt; user_name, &quot;server_threads&quot; =&gt; threads }.to_json
+            [200, {&quot;Content-Type&quot; =&gt; &quot;application/json&quot;}, [result_string]]
+          end
+          #curl &#39;localhost:8080/test?user_id=1&#39;
+</code></pre>
+</div>
```

Comment:
One thing that might be worth noting is connection management.

1. If the user wants to prepare connection beforehand (typically at startup), he can call `connect` method explicitly, otherwise the connection will be established immediately before the first command is sent
2. If the user wants to disconnect connection per request, he can call `redis.disconnect` (it's often needed if he is suffering from `Too many connection` error)

Taking the above into account, how about adding the following example with a concise explanation?
```ruby
mruby.handler: |
  redis = H2O::Redis.new(:host => '127.0.0.1', :port => 6379)
  redis.connect
  proc {|env|
    reply = redis.get('foo').join
    redis.disconnect # if you want to disconnect per request
    [200, {}, [reply]]
  }
```